embeddings/embedding/static/config.py

```diff
@@ -1,3 +1,5 @@
+import os
+import pathlib
```

Comment:
```suggestion
from pathlib import Path
```
I would just import `Path` if this script only uses `Path` from the pathlib.